mobile_verifier/src/rewarder.rs

```diff
@@ -198,6 +200,43 @@ impl Rewarder {
                 .await??;
         }
 
+        // Mapper rewards currently include rewards for discovery mapping only.
+        // Verification mapping rewards to be added
+        // Any subscriber for which the carrier has submitted a location sharing report
+        // during the epoch will be eligible for discovery mapping rewards
+
+        // get subscriber location shares this epoch
+        let location_shares =
+            subscriber_location::aggregate_location_shares(&self.pool, reward_period).await?;
+
+        // determine mapping shares based on location shares and data transferred
+        let mapping_shares = MapperShares::new(location_shares);
+        let rewards_per_share = mapping_shares.rewards_per_share(reward_period)?;
+
+        // translate discovery mapping shares into subscriber rewards
+        for mapping_share in
+            mapping_shares.into_subscriber_rewards(reward_period, rewards_per_share)
+        {
+            if self.disable_discovery_loc_rewards_to_s3 {
+                tracing::info!(
+                    "discovery location rewards output to s3 is disabled, outputting to logs only"
+                );
+                if let Some(ProtoReward::SubscriberReward(reward)) = mapping_share.reward {
+                    tracing::info!(
+                        "subscriber_id: {:?}, discovery_location_amount: {}",
+                        reward.subscriber_id,
```

Comment:
since this is just a byte array, will the debug output of the `subscriber_id` be valuable logging or will it just be visual noise? should we truncate it in case it outputs with weird formatting or base64 encode it or something?

Reply:
i think its fine as is, this is temp logging to show the discovery location rewards are being calculated in advance of them going live.  Once they are live this code should be removed